### ISSUE TITLE
Crash when opening project

### ISSUE THREAD

**Issue by cherryblossom000** [codesandbox/codesandbox-client#4170]
# 💥 Crash Report

## What were you trying to accomplish when the crash occurred?
I was trying to open [this sandbox](https://codesandbox.io/s/blueprint-sandbox-et9xy) (directly from a link). If it helps, my session had also expired and I had to sign in again.

### Crash Details

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | chrome | 81.0.4044 | OS X 10.15.3           | PROD-1583440701-41bd38c73          |

  **Route:**
  https://codesandbox.io/s/blueprint-sandbox-et9xy

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: Cannot read property 'username' of null
  TypeError: Cannot read property 'username' of null
    at ce (https://codesandbox.io/static/js/app.3ccb9c828.js:539:1584762)
    at da (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1694646)
    at Ss (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1740046)
    at Sl (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1725689)
    at xl (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1725614)
    at gl (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1722934)
    at https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1680930
    at t.unstable_runWithPriority (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1756380)
    at Go (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1680640)
    at Xo (https://codesandbox.io/static/js/common.80a8af82d.chunk.js:1:1680876)
  
    in ce
    in div
    in t
    in Actions___StyledStack
    in pr
    in header
    in t
    in Header___StyledStack
    in lo
    in t
    in Unknown
    in div
    in t
    in elements__Container
    in t
    in Unknown
    in Cv
    in t
    in t
    in div
    in t
    in elements__Content
    in t
    in withRouter(t)
    in div
    in t
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in t
    in l
    in t
  ```

  </details>

**Comment by lbogdan** [comment]
`user` is `null` here:

https://github.com/codesandbox/codesandbox-client/blob/66714e552553a38475a9386e34b281a201dbddb1/packages/app/src/app/pages/common/UserMenu/index.tsx#L48-L53

**Comment by lbogdan** [comment]
Closing this, as it should already by fixed by #3659 .
